styles/utils/flex-formats.css

```diff
@@ -21,4 +21,10 @@
 .flex-space-btwn{
     display:flex;
     justify-content: space-between;
+}
+
+.flex-col-space-btwn{

```

Comment:
Just a suggestion - since you have already created a class `flex-layout` specifically for`display: flex` , you can create further flex classes without `display: flex`, right?  like -
```
.flex-center{
    align-items:center;
    justify-content:center;
}

.flex-space-btwn{
    justify-content: space-between;
}
```

So that in your HTML you can do something like 
```html
<div class="flex-layout flex-center flex-space-btwn"></div>
```

Reply:
Yes I though of doing that, but the number of classes I had to call under 'class' grew so much and I wasn't sure about that. For example in case of shadow card I already have card-container and shadow-card classes to call. So spliting the flex made me add 3 more classes to it.

```
<!--individual flex classes-->
<div class="flex-col-space-btwn card-container shadow-card"></div>
```

```
<!--broken down flex classes-->
<div class="flex-col-layout flex-center flex-space-btwn card-container shadow-card">
```
But I'll make sure to reduce the number of lines repeated here on 👍🏽

Reply:
Sure. I get into that dilemma sometimes as well. Also, why is the shadow class specific to the card? `you can just add `shadow` and use it everywhere to get the consistency, right?